### ISSUE TITLE
random-label throws Undefined property: stdClass::$visibleoncoursepage

### ISSUE THREAD

**Issue by nordlolek** [tmuras/moosh#190]
- moosh version: 0.24
- moodle version: 3.3+
    - database: mysql  Ver 15.1 Distrib 5.5.56-MariaDB, for Linux (x86_64) using readline 5.1
- php version: PHP 5.6.30 (cli) (built: Jan 19 2017 08:09:42)
- operating system: 2.6.32-696.1.1.el6.x86_64 (centos 6)

### Actual behaviour
When I run: sudo -u apache php moosh random-label 20
I get:

```
PHP Notice:  Undefined property: stdClass::$visibleoncoursepage in *moodle/course/modlib.php on line 63

Notice: Undefined property: stdClass::$visibleoncoursepage in *moodle/course/modlib.php on line 63
Database transaction aborted automatically in *moodle
Default exception handler: Error writing to database Debug: Column 'visibleoncoursepage' cannot be null
INSERT INTO mdl_course_modules (course,module,instance,visible,visibleoncoursepage,visibleold,groupmode,groupingid,availability,showdescription,added) VALUES(?,?,?,?,?,?,?,?,?,?,?)
[array (
  0 => '20',
  1 => 12,
  2 => 0,
  3 => '1',
  4 => NULL,
  5 => '1',
  6 => 0,
  7 => 0,
  8 => NULL,
  9 => 0,
  10 => 1498808798,
)]
Error code: dmlwriteexception
* line 489 of /lib/dml/moodle_database.php: dml_write_exception thrown
* line 1259 of /lib/dml/mysqli_native_moodle_database.php: call to moodle_database->query_end()
* line 1305 of /lib/dml/mysqli_native_moodle_database.php: call to mysqli_native_moodle_database->insert_record_raw()
* line 834 of /course/lib.php: call to mysqli_native_moodle_database->insert_record()
* line 108 of /course/modlib.php: call to add_course_module()
* line 74 of /opt/moosh/Moosh/Command/Moodle23/Dev/RandomLabel.php: call to add_moduleinfo()
* line 285 of /opt/moosh/moosh.php: call to Moosh\Command\Moodle23\Dev\RandomLabel->execute()

!!! Error writing to database !!!
```

### Expected behaviour
Command should run without error.

### Steps to reproduce
Go to installed moodle 3.3 instance and run command: sudo -u apache php moosh random-label 20

### course-enrolleduser  command question
Also 'moosh course-enrolleduser teacher 8' is not found. Is this function deprecated ? What would alternative command be?
When I run command, I get:

```
moosh version 0.24
No command provided, possible commands:
	category-export
	course-backup
  [...]
```

I use these commands as a part of smoketest which I use before making production upgrade. 

Thank you in advance!

**Comment by tmuras** [comment]
Hi and thanks for reporting.
random-label is now fixes in git repo to work with Moodle 3.3.

I have deprecated & removed course-enrolleduser some time ago. Try to use moosh user-list instead,

**Comment by nordlolek** [comment]
Thank you! It's working now
